scripts/mirror_file.py

```diff
@@ -118,7 +118,7 @@ def file_part(part_number: int) -> str:
 def main(argv):
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=AzulArgumentHelpFormatter)
-    parser.add_argument('-c', '--catalog')
+    parser.add_argument('-c', '--catalog', default=config.default_catalog)
```

Comment:
```suggestion
    parser.add_argument('-c', '--catalog',
                        default=config.default_catalog,
                        help='The name of the catalog to mirror bundles from.')
```

Adding a `help` message to the field will let the user see that the field has a default, and what the default is.

Without `help='...'`
```
options:
  -h, --help                     show this help message and exit
  -c CATALOG, --catalog CATALOG
  --no-wait
```

With `help='...'`
```
options:
  -h, --help                     show this help message and exit
  -c CATALOG, --catalog CATALOG  The name of the catalog to mirror bundles from. (default: dcp3)
  --no-wait
```